pkg/network/state_test.go

```diff
@@ -718,14 +720,16 @@ func TestRaceConditions(t *testing.T) { //nolint:revive // TODO fix revive unuse
 	genConns := func(n uint32) []ConnectionStats {
 		conns := make([]ConnectionStats, 0, n)
 		for i := uint32(0); i < n; i++ {
+			sourcePort := nettestutil.GetOpenPortTCP(t)
```

Comment:
I thought we were not modifying the benchmarks?

Reply:
Also it doesn't look like any actual connections are being made here so using `rand` here should be fine.

Reply:
Ah thanks for catching. I forgot save my changes for this file. I'll also switch this to use `rand` as well.